### DISCUSSION TITLE
Chat window phone number validation

### DISCUSSION THREAD

**Discussion by AnushaPaiH** [LiveHelperChat/livehelperchat#1941] [Q&A] [answered]
Hello,  
In my website to start the live chat i am collecting name and contact number, is it possible to validate the mobile number? only number

**Comment by remdex** [comment, marked as answer]
Use the bot as it has validation capabilities.
https://doc.livehelperchat.com/docs/bot/collect-custom-attribute

And if you talking about standard start chat it does not have phone number validation, which would be needed to develop.
